sonaranalyzer-dotnet/rspec/cs/S3949_c#.html

```diff
@@ -1,6 +1,7 @@
 <p>Numbers are infinite, but the types that hold them are not. Each numeric type has hard upper and lower bounds. Try to calculate or assign numbers
 beyond those bounds, and the result will be a value that has silently wrapped around from the expected positive value to a negative one, or vice
 versa.</p>
+<p>This rule is only available for analyzers running on Windows machines. It will not be reported by analazyers running on any other OS.</p>
```

Comment:
Spelling error, I modified the rspec into, maybe update it here too.
```suggestion
<p>This rule is only available for analyses run on Windows machines. It will not be reported in analyses run on any other OS.</p>
```

Reply:
I could not see any spelling mistake there neither did Ann. 
But OK for your version,